infra_tooling_account/app.py

```diff
@@ -58,4 +58,19 @@
 alerting_stack.add_dependency(common_stack)
 monitoring_stack.add_dependency(common_stack)
 
+if all(setting is not None for setting in settings.get_grafana_mandatory_settings()):
+    grafana_stack = InfraToolingGrafanaStack(
+        app,
+        f"cf-{PROJECT_NAME}-InfraToolingGrafanaStack-{STAGE_NAME}",
+        tags=TAGS,
+        stage_name=STAGE_NAME,
+        project_name=PROJECT_NAME,
+        settings=settings,
+        env={
+            "region": os.getenv('CDK_DEFAULT_REGION'),
```

Comment:
If I'm not wrong - this requires CDK_DEFAULT_REGION env var to be set, right?

Maybe, then, 
current_region = Stack.of(self).region
looks as a more universal approach, as we let CDK to decide how to get current region.

Reply:
unfortunately, Stack.of(self).region does not work for me, I'll try to investigate further on this

Reply:
what is the error you are getting? Let's have a follow-up discussion on Jan, 3rd :)

Reply:
updated as discussed